src/cli/opts.rs

```diff
@@ -309,6 +316,34 @@ pub enum OfferSelectorParseError {
     Invalid,
 }
 
+#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Debug, Display, From)]
+pub enum CheckpointSelector {
+    #[display("All")]
```

Comment:
This is equivalent to struct level annotation `display(Debug)`

Reply:
590da5e4a8c9501c44b655891810e3c871f1e212